common/src/main/java/com/wynntils/functions/StatisticFunctions.java

```diff
@@ -0,0 +1,150 @@
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return DEFAULT_VALUE;
+
+            return statistic.total();
+        }
+    }
+
+    public static class StatisticsCountFunction extends StatisticFunction<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return DEFAULT_VALUE;
+
+            return statistic.count();
+        }
+    }
+
+    public static class StatisticsMinFunction extends StatisticFunction<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return DEFAULT_VALUE;
+
+            return statistic.min();
+        }
+    }
+
+    public static class StatisticsMaxFunction extends StatisticFunction<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return DEFAULT_VALUE;
+
+            return statistic.max();
+        }
+    }
+
+    public static class StatisticsAverageFunction extends StatisticFunction<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return DEFAULT_VALUE;
+
+            return statistic.average();
+        }
+    }
+
+    public static class StatisticsFirstModifiedFunction extends StatisticFunction<Long> {
+        @Override
+        public Long getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return 0L;
+
+            return statistic.firstModified();
+        }
+    }
+
+    public static class StatisticsLastModifiedFunction extends StatisticFunction<Long> {
+        @Override
+        public Long getValue(FunctionArguments arguments) {
+            StatisticEntry statistic = getStatisticEntry(arguments);
+            if (statistic == null) return 0L;
+
+            return statistic.lastModified();
+        }
+    }
+
+    public static class StatisticsFormattedFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            String statisticKey = arguments.getArgument("statisticKey").getStringValue();
+            int value = arguments.getArgument("value").getIntegerValue();
+
+            StatisticKind statisticKind;
+            try {
+                statisticKind = StatisticKind.from(statisticKey);
+            } catch (IllegalArgumentException e) {
+                return "-";
+            }
```

Comment:
I don't think this is working as intended
![image](https://github.com/Wynntils/Artemis/assets/8337467/497f66ea-aab2-4c79-a1cf-7e956a2ab0d5)

Reply:
Actually, it's probably because I stuck a different stat into the 2nd argument, this try catch should work but one should be needed for the `getFromattedValue`

Reply:
I forgot to do the null check after I changed from using valueOf to from.